NotebookAnalyzers/utils.py

```diff
@@ -27,7 +27,19 @@ def notebookToCode(filename):
     return py_code
 
 def functionsNumber(code):
-    """The function takes a python code sintax tree and returns the number of function definitions"""
+    """The function takes a python code and returns the number of function definitions"""
     tree = ast.parse(code)
     f_num=sum(isinstance(exp, ast.FunctionDef) for exp in tree.body)
-    return f_num
\ No newline at end of file
+    return f_num
+
+def notExecutedCells(notebook):
+    """The function takes the notebook dictionary and returns both the number empty cells in first position and the number of not executed cells in second position"""
```

Comment:
The function takes **a dict representing a** notebook and returns **a tuple with** the number empty cells as the first element and the number of **non-executed code** cells as the second element.